domain/access/bootstrap/bootstrap.go

```diff
@@ -91,6 +93,7 @@ func AddUserWithPassword(name string, password auth.Password, access permission.
 				ctx, tx,
 				uuid,
 				name, name,
+				false,
```

Comment:
We need some further validation on the name here. If the user were to add a name with a non-local domain, we would have conflicting sources of truth - tag says yes; data says no.